src/prefix_match.cc

```diff
@@ -0,0 +1,39 @@
+#include "prefix_match.h"
+
+#include <algorithm>
+#include <cctype>
+
+namespace {
+    bool StartsWith(std::string_view text, std::string_view prefix) {
+      return text.find(prefix) == 0;
```

Comment:
There may be a function in utils.h to do this

Reply:
Done. And also moved `StartsWithIgnoreCase` to utils